diff --git a/node_modules/@metamask/controllers/dist/assets/CurrencyRateController.js b/node_modules/@metamask/controllers/dist/assets/CurrencyRateController.js
index fe15421..9e53b77 100644
--- a/node_modules/@metamask/controllers/dist/assets/CurrencyRateController.js
+++ b/node_modules/@metamask/controllers/dist/assets/CurrencyRateController.js
@@ -54,7 +54,8 @@ class CurrencyRateController extends BaseController_1.default {
     getCurrentCurrencyFromState(state) {
         return state && state.currentCurrency ? state.currentCurrency : 'usd';
     }
-    getPricingURL(currentCurrency, nativeCurrency, includeUSDRate) {
+    getPricingURL(currentCurrency, _nativeCurrency, includeUSDRate) {
+        var nativeCurrency = _nativeCurrency.toUpperCase() === 'XDAI' ? 'DAI': _nativeCurrency
         return (`https://min-api.cryptocompare.com/data/price?fsym=` +
             `${nativeCurrency.toUpperCase()}&tsyms=${currentCurrency.toUpperCase()}` +
             `${includeUSDRate && currentCurrency.toUpperCase() !== 'USD' ? ',USD' : ''}`);
