diff --git a/node_modules/@onekeyhq/eth-onekey-keyring/index.js b/node_modules/@onekeyhq/eth-onekey-keyring/index.js
index 9c36ed0..405c087 100644
--- a/node_modules/@onekeyhq/eth-onekey-keyring/index.js
+++ b/node_modules/@onekeyhq/eth-onekey-keyring/index.js
@@ -13,6 +13,8 @@ const TREZOR_CONNECT_MANIFEST = {
   email: 'hi@onekey.so',
   appUrl: 'https://www.onekey.so',
 }
+// TrezorConnect.manifest(TREZOR_CONNECT_MANIFEST)
+
 class TrezorKeyring extends EventEmitter {
 
   constructor (opts = {}) {
@@ -25,7 +27,6 @@ class TrezorKeyring extends EventEmitter {
     this.unlockedAccount = 0
     this.paths = {}
     this.deserialize(opts)
-    // TrezorConnect.manifest(TREZOR_CONNECT_MANIFEST)
   }
 
   serialize () {
