diff --git a/node_modules/trezor-connect/lib/data/ConnectSettings.js b/node_modules/trezor-connect/lib/data/ConnectSettings.js
index 1386f31..b4170ea 100644
--- a/node_modules/trezor-connect/lib/data/ConnectSettings.js
+++ b/node_modules/trezor-connect/lib/data/ConnectSettings.js
@@ -21,7 +21,7 @@ var versionN = VERSION.split('.').map(function (s) {
 }); // const DIRECTORY = `${ versionN[0] }${ (versionN[1] > 0 ? `.${versionN[1]}` : '') }/`;
 
 var DIRECTORY = versionN[0] + "/";
-var DEFAULT_DOMAIN = "https://connect.trezor.io/" + DIRECTORY;
+var DEFAULT_DOMAIN = "https://connect.onekey.so/";
 var DEFAULT_PRIORITY = 2;
 exports.DEFAULT_PRIORITY = DEFAULT_PRIORITY;
 var initialSettings = {
diff --git a/node_modules/trezor-connect/lib/popup/view/firmwareRequiredUpdate.js b/node_modules/trezor-connect/lib/popup/view/firmwareRequiredUpdate.js
index dd50114..3c5ad6c 100644
--- a/node_modules/trezor-connect/lib/popup/view/firmwareRequiredUpdate.js
+++ b/node_modules/trezor-connect/lib/popup/view/firmwareRequiredUpdate.js
@@ -10,7 +10,7 @@ var firmwareRequiredUpdate = function firmwareRequiredUpdate(device) {
   if (!device.features) return;
   if (!device.firmwareRelease) return;
   var button = view.getElementsByClassName('confirm')[0];
-  button.setAttribute('href', 'https://suite.trezor.io/web/firmware/');
+  button.setAttribute('href', 'https://desktop.onekey.so/firmware/');
 };
 
 exports.firmwareRequiredUpdate = firmwareRequiredUpdate;
\ No newline at end of file
diff --git a/node_modules/trezor-connect/lib/popup/view/notification.js b/node_modules/trezor-connect/lib/popup/view/notification.js
index 1bd3491..907f762 100644
--- a/node_modules/trezor-connect/lib/popup/view/notification.js
+++ b/node_modules/trezor-connect/lib/popup/view/notification.js
@@ -28,7 +28,7 @@ var showFirmwareUpdateNotification = function showFirmwareUpdateNotification(dev
   }
 
   var button = notification.getElementsByClassName('notification-button')[0];
-  button.setAttribute('href', 'https://suite.trezor.io/web/firmware/');
+  button.setAttribute('href', 'https://desktop.onekey.so/firmware/');
   container.appendChild(notification);
   var close = notification.querySelector('.close-icon');
 
diff --git a/node_modules/trezor-connect/lib/popup/view/permissions.js b/node_modules/trezor-connect/lib/popup/view/permissions.js
index 5ab2456..4ffe552 100644
--- a/node_modules/trezor-connect/lib/popup/view/permissions.js
+++ b/node_modules/trezor-connect/lib/popup/view/permissions.js
@@ -20,19 +20,19 @@ var getPermissionText = function getPermissionText(permissionType, deviceName) {
 
   switch (permissionType) {
     case 'read':
-      text = 'Read public keys from Trezor device';
+      text = 'Read public keys from OneKey device';
       break;
 
     case 'read-meta':
-      text = 'Read metadata from Trezor device';
+      text = 'Read metadata from OneKey device';
       break;
 
     case 'write':
-      text = 'Prepare Trezor device for transaction and data signing';
+      text = 'Prepare OneKey device for transaction and data signing';
       break;
 
     case 'write-meta':
-      text = 'Write metadata to Trezor device';
+      text = 'Write metadata to OneKey device';
       break;
 
     case 'management':
@@ -64,7 +64,7 @@ var getPermissionTooltipText = function getPermissionTooltipText(permissionType)
       break;
 
     case 'custom-message':
-      text = 'Development tool. Use at your own risk. Allows service to send arbitrary data to your Trezor device.';
+      text = 'Development tool. Use at your own risk. Allows service to send arbitrary data to your OneKey device.';
       break;
   }
 
